pocs/utils/images/fits.py

```diff
@@ -368,12 +368,9 @@ def write_fits(data, header, filename, logger=None, exposure_event=None):
     else:
         if logger:
             logger.debug('Image written to {}'.format(filename))
-    finally:
-        if exposure_event:
-            exposure_event.set()
```

Comment:
IIRC the reason I originally moved the event setting here was to avoid duplicating this code in all the camera class `_readout` methods, but since the refactor that introduced the `_poll_exposure` method in the camera base class this is indeed redundant, and moving it up rather than down the call chain makes more sense.